app/code/Magento/Sales/Test/Mftf/ActionGroup/AdminClickInvoiceButtonIntoOrderActionGroup.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminClickInvoiceButtonIntoOrderActionGroup">
+        <annotations>
+            <description>Click 'Invoice' button on the order view page.</description>
+        </annotations>
+
+        <click selector="{{AdminOrderDetailsMainActionsSection.invoice}}" stepKey="clickInvoiceAction"/>
+        <waitForPageLoad stepKey="waitForProductPage"/>
+        <seeInCurrentUrl url="{{AdminInvoiceNewPage.url}}" stepKey="seeOrderInvoiceUrl"/>
```

Comment:
Having an assertion in this action group is a good idea. But it's not a commonly used practice. This approach may slow down the existing test execution routine. Also, this action group will not be potentially usable in case if the invoice functionality will appear in a popup in the future. So I would suggest being ascetic here. If the action group "clicks", it should not perform extra actions that don't directly relate to the clicking.

Reply:
**Done**